man/brackets.Rd

```diff
@@ -175,12 +169,3 @@ tbl[c(4, 3, 1), c(3, 1)] # It preserves order and wanted selection
 Regarding sorting: \code{sort_at_path()} and how to understand path
 structure: \code{summarize_row_groups()}, and \code{summarize_col_groups()}.
 }
-\keyword{internal}
-\description{
-These objects are imported from other packages. Follow the links
-below to see their documentation.
-
-\describe{
-  \item{base}{\code{\link[base:Extract]{[}}, \code{\link[base:Extract]{[<-}}}
-}}
-
```

Comment:
This is the automatic reexports documentation I believe. In practice, it was generated inside this file in a non-standard way.